docs/concepts/native-interop.rst

```diff
@@ -0,0 +1,260 @@
+Native interoperability
+=======================
+
+In this document we will dive a little bit deeper into all three ways of doing 
+"native interoperability" that are available on the .NET platform. 
+
+To refresh your memory, the two main ways of doing native interoperability are:
+
+* Platform Invoke (P/Invoke)
+* COM interoperability
```

Comment:
It is intended to be general purpose, in the sense that it is generally applicable. It makes it sound like COM Interop is available everywhere .NET is, which isn't true. P/Invoke is the "one true way".